src/index.html

```diff
@@ -18,7 +18,9 @@ <h1>2048</h1>
         </div>
       </div>
 
-      <table class="game-field">
+      <div id="board"></div>
+
+      <!-- <table class="game-field">
```

Comment:
remove comments

Reply:
when nothing to merge and no space to move cells game shouldn't continue
try to click only 'right' button for example and do the same in original game https://2048game.com/

How can i do this?